logs/logs.go

```diff
@@ -33,9 +35,60 @@ type LogLine struct {
 	Labels        map[string]string `json:"labels,omitempty"`
 }
 
+func (t LogLine) GetDedupField(field string) string {
+	switch field {
+	case "message":
+		return fmt.Sprintf("msg::%s", t.Message) // TODO: return tokenized message
+	case "hash":
+		return fmt.Sprintf("hash::%s", t.Hash)
```

Comment:
What is the difference between tokenuzed message and Hash ?

Reply:
I didn't associate hash with a fingerprint. Thought it was some other metadata for a log line.